playbooks/roles/ocim/templates/ocim-screenrc

```diff
@@ -0,0 +1,13 @@
+screen
```

Comment:
Since we're here, can we add names to these windows? Like from https://github.com/open-craft/ansible-playbooks/blob/master/playbooks/roles/accounting/templates/accounting-screenrc

Reply:
Done.